ratelimits/bucket.go

```diff
@@ -76,13 +76,9 @@ func newRegIdDomainBucketKey(name Name, regId int64, orderName string) (string,
 
 // newFQDNSetBucketKey validates and returns a bucketKey for limits that use the
 // 'enum:fqdnSet' bucket key format.
-func newFQDNSetBucketKey(name Name, orderNames []string) (string, error) {
+func newFQDNSetBucketKey(name Name, orderNames []string) string {
 	id := string(core.HashNames(orderNames))
-	err := validateIdForName(name, id)
```

Comment:
We could modify the fqdnSet case in `validateIdForName()` with the following, but I'm not sure it's worth the added complexity:

```go
    var sha256Exp = regexp.MustCompile("^[a-fA-F0-9]{64}$")
    ...
	
	if sha256Exp.MatchString(id) {
		// This is likely a hashed set of unique eTLD+1 domain names.
		return nil
	}